src/lib/ProductExporter.ts

```diff
@@ -71,17 +79,28 @@ export class ProductExporter {
         Util.showSpinner('products export');
 
         let productDefinitions = await Queries.queryProduct(conn, productList);
+        Util.isBulkApi(productDefinitions) ? productDefinitions = await Queries.bulkQueryProduct(conn, productList) : null;
         let options = await Queries.queryProductOptions(conn, productList);
+        Util.isBulkApi(options) ? options = await Queries.bulkQueryProductOptions(conn, productList) : null;
         let chargesIds = await Queries.queryProductChargesIds(conn, productList);
+        Util.isBulkApi(chargesIds) ? chargesIds = await Queries.bulkQueryProductChargesIds(conn, productList) : null;
         let productAttributes = await Queries.queryProductAttributes(conn, productList);
+        Util.isBulkApi(productAttributes) ? productAttributes = await Queries.bulkQueryProductAttributes(conn, productList) : null;
         let attributeValues = await Queries.queryProductAttributeValues(conn, productList);
+        Util.isBulkApi(attributeValues) ? attributeValues = await Queries.bulkQueryProductAttributeValues(conn, productList) : null;
         let attributeDefaultValues = await Queries.queryAttributeDefaultValues(conn, productList);
+        Util.isBulkApi(attributeDefaultValues) ? attributeDefaultValues = await Queries.bulkQueryAttributeDefaultValues(conn, productList) : null;
         let attributeValueDependencies = await Queries.queryAttributeValueDependencies(conn, productList);
+        Util.isBulkApi(attributeValueDependencies) ? attributeValueDependencies = await Queries.bulkQueryAttributeValueDependencies(conn, productList) : null;
         let attributeRules = await Queries.queryAttributeRules(conn, productList);
+        Util.isBulkApi(attributeRules) ? attributeRules = await Queries.bulkQueryAttributeRules(conn, productList) : null;
         let productRelationships = await Queries.queryProductRelationships(conn, productList);
+        Util.isBulkApi(productRelationships) ? productRelationships = await Queries.bulkQueryProductRelationships(conn, productList) : null;
+
```

Comment:
Sam fakt że przez x linijek pojawia się sprawdzenie isBulkAPI sugeruje że bardziej elegancko byłoby to sprawdzenie jednak umieścić gdzieś indziej